styles/styles.css

```diff
@@ -0,0 +1,278 @@
+}
+
+::marker {
+  font-size: 14px;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 100%;
+  color: #828282;
+  font-size: 8pt;
+  font-family: Verdana, Geneva, sans-serif;
+}
+
+main {
+  width: 85%;
+  height: 100vh;
+  background: #f6f6ef;
+  display: flex;
+  flex-direction: column;
+  margin: 8px;
+}
+
+.page {
+  display: grid;
+  grid-auto-rows: max-content auto;
+  grid-template-columns: repeat(2, 1fr);
+}
+
+.page__header {
+  grid-column: 1/3;
+  display: grid;
+  justify-content: start;
+  grid-template-columns: auto auto;
+  align-items: center;
+}
+
+.page__news {
+  grid-column: 1;
+  grid-row: 2;
+}
+
+.page__form {
+  grid-column: 2;
+  grid-row: 2;
+}
+
+@media only screen and (min-width: 300px) and (max-width: 750px) {
+  main {
+    width: 100%;
+  }
+}
+
+.top-panel {
+  padding: 2px;
+  background-color: #f60;
+  margin-bottom: 10px;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: auto;
+}
+
+.logo__title {
+  color: #222;
+  line-height: 12px;
+  font-weight: bold;
+  padding-left: 2px;
+  font-size: 10pt;
+  display: flex;
+  align-items: center;
+}
+
+.logo__title::before {
+  background-image: url('../images/logo.gif');
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: left;
+  width: 18px;
+  height: 18px;
+  display: inline-flex;
+  content: '';
+  border: 1px solid white;
+}
+
+.logo__image {
+  border: 1px solid white;
+  width: 18px;
+  height: 18px;
+  background-size: contain;
+  background-image: url('../images/logo.gif');
+}
+
+.menu__item {
+  color: black;
+  font-family: Verdana, Geneva, sans-serif;
+  font-size: 10pt;
+}
+
+.news {
+  display: flex;
+  flex-direction: column;
+}
+
+.generic-list {
+  display: flex;
+  flex-direction: column;
+  padding-left: 28px;
+}
+
+.generic-list--hidden {
+  display: none;
+}
+
+.generic-list__show-comments {
+  border: 1px solid transparent;
+  background: transparent;
+  color: inherit;
+  cursor: pointer;
+}
+
+.generic-list__show-comments:hover {
+  border-bottom: 1px solid #828282;
+}
+
+.generic-list__item {
+  position: relative;
+  margin-bottom: 7px;
+  padding-left: 8px;
+}
+
+.generic-list__item::before {
+  width: 13px;
+  height: 13px;
+  background-repeat: no-repeat;
+  background-size: contain;
+  position: absolute;
+  left: -5px;
+  top: 2px;
+  content: '';
+  display: flex;
+  background-image: url('../images/arrow.gif');
+}
+
+.generic-list__news-header {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: 1rem;
+}
+
+.generic-list__detail {
+  color: #828282;
+  font-size: 8pt;
+}
+
+.generic-list__title {
+  font-size: 10pt;
+  color: #000;
+  font-weight: normal;
+}
+
+.generic-list__content {
+  color: #000;
+  font-size: 9pt;
+  font-weight: normal;
+}
+
+.form {
+  flex-direction: column;
+  display: flex;
+  border: 1px solid grey;
+  padding: 10px;
+}
+
+.form__control {
```

Comment:
```.form__control``` should be responsible for positioning only
look and feel should be placed within .field block
```.form__control {}

.field {
  margin: 3px 0;
  height: 30px;
  align-items: center;
  display: grid;
  grid-template-columns: 40% 60%;
  grid-template-rows: 1fr;
}```

Reply:
Fixed. And delete form-control class, because he do not have any styles